core/interop/source-generation/ComWrappersGeneration/Server/FileUtils.cs

```diff
@@ -0,0 +1,27 @@
+
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+
+namespace Tutorial;
+
+public static class FileUtils
+{
+    public static unsafe bool TryGetDllPath([NotNullWhen(true)]out string? dllPath)
```

Comment:
I don't think this is needed. I would just use something like `typeof(Exports).Assembly.Location`. There is no need to call any Win32 API for the path of the assembly.

Reply:
For NativeAOT we can't use Assembly.Location, so we need this. For the DNNE version we'll use that.